### DISCUSSION TITLE
Is there a way to search by Date?

### DISCUSSION THREAD

**Discussion by polt515** [gdh1995/vimium-c#713] [Q&A] [answered]
Similar to searching bookmark folders with / Is there a way to do this with a date like /Jul to see only July entries?

**Comment by gdh1995** [comment, marked as answer]
Sorry but there's no way to direct filter history by date. But when you query a prefix part of a URL, it will sort suggestions by visit time in desc order.

And Vomnibar can show visit time of history suggestions, if you add some configuration:
![image](https://user-images.githubusercontent.com/5547703/187064277-4b141e52-9504-41eb-a70a-4c4c63ddce4d.png)
